### DISCUSSION TITLE
Takım kaptanları

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/BilsemPython#56] [Duyurular]
Sınıf ve github aktivitelerine göre aşağıda belirttiğim kişiler en az 4 en fazla 6 kişiden oluşan takımlar kurmakla işe başlayıp sonra da takım çalışmalarını koordine etme ve rehberlik yapmakla yükümlüdür.

@CinarSatir @akgembi12 @KayraMertAydin @FurkanYildirm @tolga0102 @Berkek007 @adasue @AcelyaP @miraykaracol @mervedam

Öncelikle forum bölümüne yeni bir başlık açarak (kendi adınız olabilir veya istediğiniz bir takım adı) üyelerinizin başvurularını değerlendirip takımınızı oluşturmakla başlayabilirsiniz.

[Yeni bir başlık açmak için tıklayın.](https://github.com/sonsuzus/BilsemPython/discussions/new)

**Comment by sonsuzus** [comment]
10 Ocak Salı günü takım kaptanları, takımlarının üyelerini bu başlık altına yazabilir. İlk takım göreviniz 11 Ocak Çarşamba günü verilecektir. O güne kadar her hangi bir takıma üye olmayan öyg öğrencisi kalmasın.

**Reply by sonsuzus** [reply]
Ayrı bir yorum olarak yazar mısın sen de arkadaşların gibi.

**Reply by mervedam** [reply]
Yazdım hocam

**Comment by Furkanyildirm** [comment]
Calcurators takımı üyeleri;
@Furkanyildirm
@BerraAlkann
@kaanpinar04
@AdaNilPoyraz
@NecipFG
@YagizEfeZ

**Comment by AcelyaP** [comment]
KittyCats takım üyeleri;

@AcelyaP 
@slos8888 
@eyupbjk 
@NidaZeynep 
@Barlasgng

**Reply by AcelyaP** [reply]
hocam bir kişi daha var ama github adını bilmiyorum

**Reply by sonsuzus** [reply]
Janset Arslan - @yidhraz ı da takıma ekler misin

**Reply by jxnspect** [reply]
selamm suan ben takimda miyim

**Comment by KayraMertAydin** [comment]
Pasific Takım üyeleri :
@KayraMertAydin
@EfeToprak
@Barlasgng
@kadirefeyilmaz27

**Comment by CinarSatir** [comment]
Rookies Takımı Üyeleri 
@CinarSatir
@OmerMete
@OmerB2
@Efemert2010 
@Sadik1Dev
@Mertcakm

**Comment by Berkek007** [comment]
Black Rhinos Üyeleri
@Berkek007
@AhmetEmirErsoz
@Alperg123456
@onursonkaya
@OmerAkbayrak

**Comment by tolga0102** [comment]
Piton Yılanları takımı üyeleri:
@tolga0102
@Ersan0102
@Atakanoyg1
@aliilker7
@efeozceylan
@TheDemr0

**Reply by efeozceylan** [reply]
Beni ekleyebilirsin 05526165476

**Comment by miraykaracol** [comment]
Apollo takim uyeleri:
@miraykaracol
@D3r3n
Nazlı Yağmur Yıldırım
Ceylin Demirtaş

**Comment by adasue** [comment]
İyi günler Ufuk Hocam.
MONSTERs grubu üyeleri aşağıdaki gibidir:
@adasue
@ahmetsemihucar
@EfeMertIrez
@Akifatalar12
@dudusoo
@Adaisland

**Comment by mervedam** [comment]
Phoenix takım üyeleri:
@mervedam
@ElifNurSoyler
@301-415-210 veya @301-210-415
@Ardemin

**Reply by mervedam** [reply]
Hocam irem sena githup kullanıcı adını ikisinden biri dediği için ikisinide yazdım
